src/user/UserClient.ts

```diff
@@ -0,0 +1,20 @@
+import { AbstractRestfulClient, Parameters } from '../abstractRestfulClient';
+import { CompleteWhoAmICompanyData } from './types/CompleteWhoAmICompany';
+import { CompleteWhoAmI } from './types/CompleteWhoAmI';
+import { CompleteWhoAmIUserData } from './types/CompleteWhoAmIUser';
+import { GetResult } from '../getResult';
+
+export type CompleteWhoAmIResponseData = {
```

Comment:
move in src/user/types/CompleteWhoAmI.ts

Reply:
done